middlewares/contentTypeCheck.js

```diff
@@ -7,7 +7,7 @@
  * @param {function} next - Express middleware function
  */
 module.exports = (req, res, next) => {
-  if (req.headers['content-type'] && req.headers['content-type'] !== 'application/json') {
+  if (req.headers['content-type'] && (req.headers['content-type'] === 'application/json' || req.headers['content-type'] === 'multipart/form-data')) {
```

Comment:
I guess it should be `!==` , right?
I suppose that's why the tests are failing

Reply:
Have corrected it